isites_migration/utils.py

```diff
@@ -35,6 +35,23 @@
     ssl._create_default_https_context = ssl._create_unverified_context
 
 
+def get_school(course_instance_id):
+    """
+    get the school_id for the course_instance_id provided.
+    :param course_instance_id:
+    :return school:
+    """
+    school = ''
+    try:
+        ci = CourseInstance.objects.get(course_instance_id=course_instance_id)
+        school = ci.course.school_id
+    except CourseInstance.DoesNotExist as e:
+        logger.exception(u'course instance id %s does not exist: %s' % course_instance_id, e)
```

Comment:
We can still fall back on looking the course instance up by canvas course id if the course instance id lookup fails (potentially because it's not set on the canvas course for some reason).